modules/route53/main.tf

```diff
@@ -1,21 +1,33 @@
 resource aws_route53_zone "primary" {
   name = var.customer_domain
+}
 
-  lifecycle {
-    prevent_destroy = true
-  }
```

Comment:
Should this be retained? We don't want the primary domain or certificates to be destroyed.. Though I guess in this case, we're setting up temporary ones which can be destroyed. Maybe make this a variable with default `true`, and set it to false with a TODO for Cloudera?

Reply:
sure I'll set that with a variable

Reply:
I set it by default